api/controllers/builds.go

```diff
@@ -67,7 +67,14 @@ func BuildGet(rw http.ResponseWriter, r *http.Request) *httperr.Error {
 
 func BuildCreate(rw http.ResponseWriter, r *http.Request) *httperr.Error {
 	build := models.NewBuild(mux.Vars(r)["app"])
-	config := r.FormValue("config")
+	build.Description = r.FormValue("description")
+
+	manifest := r.FormValue("manifest") // empty value will default "docker-compose.yml" in cmd/build
+
+	// use deprecated "config" param if set
+	if config := r.FormValue("config"); config != "" {
```

Comment:
If somehow these both get set manifest should probably win (or it should be an error)